sdk/core/core-amqp/test/requestResponse.spec.ts

```diff
@@ -1,7 +1,7 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
 
-import * as assert from "assert";
+import { assert } from "chai";
```

Comment:
Are we sure chai exports assert?
<img width="1087" alt="Screen Shot 2020-05-17 at 5 20 50 PM" src="https://user-images.githubusercontent.com/16890566/82163941-d9695f80-9862-11ea-8936-f0248ab69e62.png">

Reply:
Ughh I needed to add this to the `rollup.base.config.js`, so that the browser build would recognize this:

`chai: ["should", "assert"],`